esp/esp/themes/theme_data/circles/less/main.less

```diff
@@ -298,12 +298,6 @@ div .error
     color: #666666;
 }
 
-#usersearchform
-{
-    border: 1px solid #999999;
-    padding: 6px;
-}
-
 #usersearchform h3
```

Comment:
For me, in the circles theme, there's a user search box in the admin bar and a user search box in the theme itself on the sidebar. I think we should probably remove the one built into circles?

Reply:
It's gone now